api/project/target/streams/runtime/internalDependencyClasspath/$global/streams/export

```diff
@@ -1 +1 @@
-/Users/matteo/Documents/Buildo/apiseed/project/target/scala-2.12/sbt-1.0/classes
+/Users/matteo/Documents/Buildo/cryptonite/api/project/target/scala-2.12/sbt-1.0/classes
```

Comment:
Generalmente tutta la roba sotto /target/ va in gitignore... come mai è committata?